### PR TITLE
19926 certificateOfNameCorrection

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#19926

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/reports/report.py [hunk 1]
```diff
@@ -1313,6 +1313,10 @@ class ReportMeta:  # pylint: disable=too-few-public-methods
             'filingDescription': 'Certificate of Name Change',
             'fileName': 'certificateOfNameChange'
         },
+        'certificateOfNameCorrection': {
+            'filingDescription': 'Certificate of Name Correction',
+            'fileName': 'certificateOfNameChange'
+        },
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/reports/report.py:1319]: Do you have a UI example of this?

Comment (severinbeauvais) [reply legal-api/src/legal_api/reports/report.py:1319]: Does the list of outputs look any different in Filings UI?

Comment (severinbeauvais) [reply legal-api/src/legal_api/reports/report.py:1319]: It is intentional that the fileName is the same as certificate of name change?

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/reports/report.py:1319]: filieName you see here is the filie name of the html template (using the same template) not the output file name

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/reports/report.py:1319]: image

Comment (severinbeauvais) [reply legal-api/src/legal_api/reports/report.py:1319]: If you think the output name and link (in Filings UI) work fine, go ahead and resolve this conversation.

**Review by severinbeauvais — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**